buddyboss/buddyboss-platform
# PROD-766 Fix #2776 Notification issue > Wrong name displayed when the user reply to reply in the discussion

Notification issue > Wrong name displayed when the user reply to reply in the discussion
* Updated the secondary_item_id argument to $secondary_item_id
* Remove extra condition $author_id !== $topic_author_id

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2776 .

### How to test the changes in this Pull Request:

1. Go to 'forum'
2. Click on 'any discussion' from any user say A (who created the discussion)
3. Create a reply from another user say B
4. Create a reply to the reply created at above step 3 by another user say C ( will receive the notification to user B with the correct name of user C)
5. Also check create a reply to the reply created at above step 3 by same user say A who created the discussion( will now receive the notification to user B )

### Proof Screenshots or Video

http://somup.com/crQQfmruGx

### Other information:




### Changelog entry

> Updated the secondary_item_id argument to $secondary_item_id & remove extra condition $author_id !== $topic_author_id

**App Native Issue**
https://github.com/buddyboss/buddyboss-app-native/issues/1874
